kvdb/backend.go

```diff
@@ -260,7 +260,17 @@ func GetTestBackend(path, name string) (Backend, func(), error) {
 			_ = f.DB().Close()
 		}, nil
 
-	case TestBackend == BoltBackendName:
+	case EtcdBackend:
```

Comment:
Can we add a comment above the `switch` statement explaining that for the tests we _expect_ only one build tag to be selected at the same time? This confused me quite a bit because for the production build we turn on all three build tags at the same time.